cfme/tests/containers/test_containers_smartstate_analysis.py

```diff
@@ -68,9 +93,9 @@ def test_containers_smartstate_analysis(provider, test_item, soft_assert):
                     'Traceback: \n{}'.format(format_exc()))
 
     evm_tail.fix_before_start()
-
     image_obj.perform_smartstate_analysis(wait_for_finish=True)
-
+    #  evm_tail.validate_logs()
+    time.sleep(5 * 60)  # temp debug!
```

Comment:
Is there a BZ associated with this? I'd put a TODO or FIXME on it.  In your tests folder, up to you.